build/devops_gated.yml

```diff
@@ -290,3 +293,5 @@ jobs:
       filePath: './build/linux/build_linux_iwyu.sh'
       arguments: '$(Build.Repository.LocalPath)'
       workingDirectory: '$(Build.Repository.LocalPath)'
+
+  - template: templates/dump_drive_usage.yml
```

Comment:
So this build YML file is different than block store because it's a single file with multiple jobs. What we want to do with this "dump drive usage" is to dump it when a job fails. And specifically, when a job fails on Windows (it's not setup for Linux though likely it could be made to run on Linux as well).

You've added it to the `includecheck_linux` job.

Instead you should add it to:
- `windowsx64debug`
- `windowsx64relwithdebinfo`

And maybe `Traceability`